### DISCUSSION TITLE
Data security

### DISCUSSION THREAD

**Discussion by vtnhan1309** [milvus-io/milvus#35405] [Q&A and General discussion] [answered]
# Does Milvus encrypt vector data ?
As far as I know, we can protect our vector data by enabling security options in MinIO. But on local volume mode, does Milvus self-encrypt data before it saving them on local volume (disk) ?

**Comment by xiaofan-luan** [comment]
no we didn't do that

**Reply by vtnhan1309** [reply]
So, Can I protect my data by using Milvus with MinIO ?

**Comment by xiaofan-luan** [comment]
it's not gonna to be super hard to implement it but this is not the priority for now

**Comment by yhmo** [comment, marked as answer]
Milvus doesn't encrypt data.
MinIO Server-Side Encryption (SSE) might meet your requirements.
https://min.io/docs/minio/kubernetes/upstream/operations/server-side-encryption.html
